Backend/router/v1/report.ctrl.js

```diff
@@ -49,9 +49,31 @@ const deleteReport = (req, res) => {
   }).catch((err) => {});
 }
 
+const editReport = (req, res) => {
+  const reportId = parseInt(req.params.reportId);
```

Comment:
in negative case, what does parseInt return?
For example, if req.params.reportId passed with some garbage string, what is going to happen?

Reply:
@mtnviewlab  We are trying to unify between user part and report part, so I change this to using query string.

Reply:
@mtnviewlab  I added if statement! Thank you.